extensions/wikia/UserRenameTool/UserRenameToolHelper.class.php

```diff
@@ -63,23 +58,23 @@ static public function lookupRegisteredUserActivity( $userID ) {
 	/**
 	 * Gets wikis an IP address might have edits on
 	 *
-	 * @author Daniel Grunwell (Grunny)
-	 *
 	 * @param String $ipAddress The IP address to lookup
-	 *
 	 * @return array
 	 */
 	public static function lookupIPActivity( $ipAddress ) {
 		global $wgDevelEnvironment, $wgSpecialsDB;
+
 		wfProfileIn( __METHOD__ );
```

Comment:
If we're cleaning up, we can remove these profiling statements.  We have Xdebug and influxdb to handle this for us.